packages/backend-test-utils/CHANGELOG.md

```diff
@@ -897,7 +897,7 @@
 - cc4228e: Switched module ID to use kebab-case.
 - b7de76a: Added support for PostgreSQL versions 15 and 16
 
-  Also introduced a new `setDefaults(options: { ids?: TestDatabaseId[] })` static method that can be added to the `setupTests.ts` file to define the default database ids you want to use throughout your package. Usage would look like this: `TestDatabases.setDefaults({ ids: ['POSTGRES_12','POSTGRES_16'] })` and would result in PostgreSQL versions 12 and 16 being used for your tests.
+  Also introduced a new `setDefaults(options: { ids?: TestDatabaseId[] })` static method that can be added to the `setupTests.ts` file to define the default database ids you want to use throughout your package. Usage would look like this: `TestDatabases.setDefaults({ ids: ['POSTGRES_13','POSTGRES_17'] })` and would result in PostgreSQL versions 12 and 16 being used for your tests.
```

Comment:
Same here, we should not be updating the changelogs like this retroactively. Can you please revert this change?